index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```

Comment:
- [ ] Maintain indentation throughout the project. I'd recommend to use 2 spacs.